src/pagination/pagination.nav.stories.ts

```diff
@@ -0,0 +1,91 @@
+import {
```

Comment:
The filename should probably be `pagination-nav.stories.ts` (dash instead of dot in the base name)